packages/nimble-components/src/dialog/styles.ts

```diff
@@ -15,7 +15,8 @@ import {
     elevation3BoxShadow,
     dialogSmallWidth,
     dialogSmallHeight,
```

Comment:
I was surprised not to see Chromatic diffs for the styling change.

Is it because the focus visible state only happens if you open the dialog from a focus visible element (i.e. by focusing the button and hitting enter)? If so should we add a matrix test that does that in a play function?

Or is it a gap in how Chromatic takes snapshots? I noticed [Malcolm added a workaround to one of his PRs for something that seems related](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/652640?_a=files&path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/tests/angular-material/angular-material-host.component.scss) so you could talk to him about copying that pattern.

It's not critical to add test coverage since we have lots of existing gaps for focus visible chromatic testing, but if it's easy then let's do it.

Reply:
The only way I know of to get the dialog in a focused state is to set `prevent-dismiss`, make sure nothing in the dialog has focus, then hit ESC.

We should add a focus-state visual test once I get my Innovation Day changes submitted to the Storybook pseudo-states addon. Until then, I think we're fine without it.